components/list/demo/list-color-indicator.html

```diff
@@ -0,0 +1,336 @@
+<!DOCTYPE html>
```

Comment:
I created a demo page just to make it easier to see changes during development. If this is too much clutter in the demo pages I can remove this one post-dev and just add color indicators throughout the other pages.